index.html

```diff
@@ -16,6 +16,7 @@
     <link rel="stylesheet" href="./Components/image/image.css">
     <link rel="stylesheet" href="./Components/input/input.css">
     <link rel="stylesheet" href="./Components/navbar/navbar.css">
+    <link rel="stylesheet" href="./Components/grid/grid.css">

```

Comment:
Instead of linking every CSS file, import all CSS files in one and then link that one CSS file into your HTML file.